src/views/Export/ExportTypePane/index.js

```diff
@@ -45,49 +65,43 @@ export default class ExportTypePane extends React.PureComponent {
         nodes: level.sublevels && ExportTypePane.mapReportLevelsToNodes(level.sublevels),
     }));
 
-    static createReportStructure = (analysisFramework) => {
-        if (!analysisFramework) {
-            return undefined;
-        }
-
-        const { exportables, widgets } = analysisFramework;
-        const nodes = [];
+    // NOTE: This function generates dimension first level
+    static transformMatrix2dLevels = ({
```

Comment:
Why not use simple argument list, sir?

Reply:
Didn't want to separate it where the function was being used.
Currently it takes data of widget (object) as its param